packages/common/src/store/dsl-workflow.ts

```diff
@@ -132,6 +132,12 @@ export interface DSLActivitySpec<PARAMS extends Record<string, any> = Record<str
      */
     await?: string; //the activity name to await
 
+    /**
+     * Acitity options for configuring the activity execution.
+     *
+     * This must be an ActivityOptions from @temporalio/common //TODO: why not type it this way?
+     */
+    options?: Record<string, any>;
```

Comment:
note: this uses the same type as the DSLWorkflowSpec's `options` to be consistent, but I can use the real type as well 

https://github.com/becomposable/composableai/blob/e131c6fb2a9a2cd1f2240e9a5b033ae85b1902fd/packages/common/src/store/dsl-workflow.ts#L146-L147

Reply:
Discussed with @bstefanescu , I am going to change the type and use a custom ActivityOptions